src/main/scala/za/co/absa/hyperdrive/trigger/persistance/DagInstanceRepository.scala

```diff
@@ -83,5 +85,12 @@ class DagInstanceRepositoryImpl extends DagInstanceRepository {
     )
   }
 
+  override def hasInQueueDagInstance(workflowId: Long)(implicit executionContext: ExecutionContext): Future[Boolean] = {
+    db.run(
+      dagInstanceTable.filter(dagInstance =>
+        dagInstance.workflowId === workflowId && dagInstance.status.inSetBind(Set(DagInstanceStatuses.InQueue))
```

Comment:
Couldn't make this work with `===` instead of `inSetBind` with single element set. Do you have an idea?

Reply:
I would do it same way. 
But you can do it this way `dagInstance.status === (DagInstanceStatuses.InQueue: DagInstanceStatus)`

Reply:
Thanks a lot! But if you don't mind, then I'll leave it as it is.